interface-definitions/service_monitoring_node_exporter.xml.in

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="service">
+    <children>
+      <node name="monitoring">
+        <children>
+          <node name="node-exporter" owner="${vyos_conf_scripts_dir}/service_monitoring_node-exporter.py">
+            <properties>
+              <help>Prometheus exporter for hardware and operating system metrics</help>
+              <priority>1280</priority>
+            </properties>
+            <children>
+              #include <include/listen-address.xml.i>
+              <leafNode name="listen-address">
+                <defaultValue>0.0.0.0</defaultValue>
```

Comment:
Probably should default to `::` so IPv6 scrapes work too

Reply:
with 0.0.0.0 the exporter listen on *:9100 and so also on all IPv6 addresses:

```
vyos@vyos:~$ show configuration commands | grep node-exporter
set service monitoring node-exporter

vyos@vyos:~$ sudo ss -ltnp | grep 9100
LISTEN 0      4096               *:9100            *:*    users:(("node_exporter",pid=2336,fd=3))

vyos@vyos:~$ curl http://[2003:fa:af09:1d00:be24:11ff:fe99:28cb]:9100/metrics | head
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0# HELP go_gc_duration_seconds A summary of the pause duration of garbage collection cycles.
# TYPE go_gc_duration_seconds summary
go_gc_duration_seconds{quantile="0"} 2.7739e-05
go_gc_duration_seconds{quantile="0.25"} 2.7739e-05
go_gc_duration_seconds{quantile="0.5"} 6.2428e-05
go_gc_duration_seconds{quantile="0.75"} 6.2428e-05
go_gc_duration_seconds{quantile="1"} 6.2428e-05
go_gc_duration_seconds_sum 9.0167e-05
go_gc_duration_seconds_count 2
# HELP go_goroutines Number of goroutines that currently exist.
```

Reply:
Unintuitive but fair enough

Reply:
If the daemon listens on all interfaces if listen address is not given, maybe just make it optional and don't give it any default?